backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-recommended-network-settings.adoc"]
+= {oadp-short} recommended network settings
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+:context: oadp-recommended-network-settings
+
+toc::[]
+
+For a supported experience with {oadp-first}, you should have a stable and resilient network across {OCP-short} nodes, S3 storage, and in supported cloud environments that meet {OCP-short} network requirement recommendations.
+
+To ensure successful backup and restore operations for deployments with remote S3 buckets located off-cluster with suboptimal data paths, Red Hat recommends that your network settings meet the following minimum requirements in such less optimal conditions:
```

Comment:
I think we prefer to use _it is recommended_ in such situations (as awkward as it sounds to me).
```suggestion
To ensure successful backup and restore operations for deployments with remote S3 buckets located off-cluster with suboptimal data paths, it is recommended that your network settings meet the following minimum requirements in such less optimal conditions:
```